packages/client/src/components/interface/DownloadButton.tsx

```diff
@@ -48,18 +48,16 @@ const StatusIndicator = styled.div<{
   isLoading?: boolean
 }>`
   display: flex;
-  flex-grow: 1;
+  flex-grow: 0;
```

Comment:
Why 0 and in core component?

Reply:
Agreed but lets merge and then the coming UI dev can move DownloadButton into comp lib